### DISCUSSION TITLE
POST /api/v1/challenge/reset/38970833402/AsdvYB0cNV/ - 400 Bad Request; challenge_required

### DISCUSSION THREAD

**Discussion by vpaflah** [officialputuid/toolsig#29] [General]
When I tried to login, an error is showing like this " POST /api/v1/challenge/reset/38970833402/AsdvYB0cNV/ - 400 Bad Request; challenge_required" and exiting the program

**Comment by officialputuid** [comment]
Check ur IG, and acc verification question will appear, and select "it was me", then re-login to the tools.

**Reply by vpaflah** [reply]
Thank you for the answer.
